src/app/report/report.component.html

```diff
@@ -1,8 +1,21 @@
-<div class="report-tab row h-100">
-  <div class="resizable-horizontal w-25 p-2 border-right h-100">
-    <jqxTree #treeReference (onItemClick)="selectReport($event)" [auto-create]="false"></jqxTree>
-  </div>
-  <div class="col" *ngIf="reportData">
-    <app-edit-display (saveReportEvent)="savingReport($event)"></app-edit-display>
-  </div>
-</div>
+<as-split #splitter direction="horizontal">
+
+  <as-split-area [size]="25">
+    <div class="h-100">
+      <app-debug-tree
+        [adjustWidth]="treeWidth.asObservable()"
+        (closeEntireTreeEvent)="closeEntireTree()"
+        (selectReportEvent)="selectReport($event)"
+        [currentView]="currentView"></app-debug-tree>
+    </div>
+  </as-split-area>
+  <as-split-area [size]="75" style="overflow: hidden">
+    <app-edit-display
+      [newTab]="newTab"
+      [containerHeight]="calculatedHeight"
+      (closeReportEvent)="closeReport($event)"
+      [currentView]="currentView"
+      [id]="'debug'"></app-edit-display>
```

Comment:
this one doesn't need to be bound

Reply:
This was a copy paste, but I'll change it.